index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
The fact that these _very_ important conversations are happening within review comments of a pull request is a pretty good sign that this approach to spec writing isn't an adequate one.

_The role of the editor is to gather consensus from the WG and document it in a spec._ It is absolutely clear here that there is no such consensus. And it's really bothering that these conversations are buried in here.

I strongly suggest discussing and gathering higher-level consensus around these issues before diving in editorial work. Addressing the open issue on this topic, such as the email I penned on the mailing list in defence of writing a single motion sensor spec, seems like the right way to proceed.